### PR TITLE
feat: factor out translations

### LINKED ISSUE #46: factor out strings
In order to use graasp/graasp-translations.

### PR DESCRIPTION
**Pull request by juancarlosfarah**
Closes #46

### CODE REVIEW

File: src/components/sections/About.js [hunk 1]
```diff
@@ -140,44 +128,33 @@ const About = () => {
...
+                    components={[
+                      <a
+                        href="https://insights.graasp.org"
+                        target="_blank"
+                        rel="noopener noreferrer"
+                      >
+                        Graasp Insights
+                      </a>,
+                      <a
+                        href="https://library.graasp.org"
```
Comment (pyphilia) [inline src/components/sections/About.js:149]: Should we factor out these links?

File: src/components/common/SEO.js [hunk 2]
```diff
@@ -1,20 +1,28 @@
...
+      t(ASSOCIATION.SEO_KEYWORD_GRAASP_ASSOCIATION),
+      t(ASSOCIATION.SEO_KEYWORD_DIGITAL_EDUCATION),
+      'Graasp',
+      'EPFL',
+    ],
+    img:
+      'https://images.unsplash.com/photo-1504860708171-19abd233ec3e?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1050&q=80',
+    twitterId: 'graasp',
+    facebookId: 'graasp',
+  };
```
Comment (pyphilia) [inline src/components/common/SEO.js:24]: Should we factor out some data?

File: src/components/sections/Faq.js [hunk 3]
```diff
@@ -12,92 +12,90 @@ import { useTranslation, Trans } from 'react-i18next';
...
+            <Trans
+              components={[
+                <a
+                  href="https://graasp.org"
+                  target="_blank"
+                  rel="noopener noreferrer"
+                >
+                  Graasp
+                </a>,
+                <a href="mailto: contact@graasp.org">contact@graasp.org</a>,
```
Comment (pyphilia) [inline src/components/sections/Faq.js:82]: Factor out the email address?

File: src/config/i18n/fr.json [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  "TITLE_COMMUNITY_MANAGER": "Animateur de communauté",
+  "TITLE_AMBASSADOR": "Ambassadeur",
+
+  "HEADER_ABOUT": "À propos",
+  "HEADER_PARTNERS": "Nos solutions sont utilisées dans le monde entier",
+  "HEADER_TEAM": "L'Équipe",
+  "HEADER_FAQ": "Foire aux questions",
+  "HEADER_SUPPORTED_BY": "L'association Graasp est soutenue par",
+
+  "QUESTION_PLATFORMS": "Quelles sont les plates-formes de Graasp?",
```
Comment (pyphilia) [inline src/config/i18n/fr.json:36]:
```suggestion
"QUESTION_PLATFORMS": "Quelles sont les plateformes de Graasp?",
```

File: src/config/i18n/fr.json [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+  "ANSWER_PLATFORMS": "La solution phare de l'Association Graasp est <0>Graasp</0>, une plate-forme d'éducation numérique ouverte accessible sur le Web, les appareils mobiles et les ordinateurs de bureau, intégrant une suite d'applications d'apprentissage.",
```
Comment (pyphilia) [inline src/config/i18n/fr.json:37]:
```suggestion
"ANSWER_PLATFORMS": "La solution phare de l'Association Graasp est <0>Graasp</0>, une plateforme d'éducation numérique ouverte accessible sur le Web, les appareils mobiles et les ordinateurs de bureau, intégrant une suite d'applications d'apprentissage.",
```

File: src/config/i18n/fr.json [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+  "QUESTION_GRAASP": "Qu’est-ce que Graasp?",
+  "ANSWER_GRAASP": "<0>Graasp</0> est une plate-forme sur laquelle vous pouvez créer vos propres activités ou ressources pédagogiques et numériques. Pour vous aider à démarrer, Graasp fournit des modèles de scénarios d'apprentissage que vous pouvez adapter à vos besoins. Vous pouvez soit partager ces ressources de manière privée avec vos élèves soit les mettre à disposition de quelques collègues ou de tous.",
```
Comment (pyphilia) [inline src/config/i18n/fr.json:39]:
```suggestion
"ANSWER_GRAASP": "<0>Graasp</0> est une plateforme sur laquelle vous pouvez créer vos propres activités ou ressources pédagogiques et numériques. Pour vous aider à démarrer, Graasp fournit des modèles de scénarios d'apprentissage que vous pouvez adapter à vos besoins. Vous pouvez soit partager ces ressources de manière privée avec vos élèves soit les mettre à disposition de quelques collègues ou de tous.",
```

File: src/config/i18n/fr.json [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+  "QUESTION_GETTING_STARTED": "Comment puis-je commencer?",
+  "ANSWER_GETTING_STARTED": "Vous pouvez essayer nos plates-formes gratuitement en vous inscrivant <0>ici</0>.",
```
Comment (pyphilia) [inline src/config/i18n/fr.json:41]:
```suggestion
"ANSWER_GETTING_STARTED": "Vous pouvez essayer nos plateformes gratuitement en vous inscrivant <0>ici</0>.",
```

File: src/config/i18n/fr.json [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+  "QUESTION_COST": "Combien ça coûte?",
+  "ANSWER_COST": "Vous pouvez commencer à utiliser <0>Graasp</0> gratuitement ! L'accès aux fonctionnalités premium, telles que les analytiques d'apprentissage et les espaces d'apprentissage personnalisés, coûte 5 CHF par mois, pour chaque utilisateur enregistré. Les institutions peuvent bénéficier de licences groupées ou d’hébergement sur site (en préparation) pour leurs enseignant-e-s. Si vous utilisez Graasp pour des activités à but non lucratif, vous pouvez bénéficier d’un soutien financier. Pour davantage d’informations, veuillez nous contacter à <1>contact@graasp.org</1>.",
+  "QUESTION_TRAINING": "Est-ce que vous proposez des formations?",
```
Comment (pyphilia) [inline src/config/i18n/fr.json:44]:
```suggestion
"QUESTION_TRAINING": "Est-ce que vous proposez des formations ?",
```

File: src/config/i18n/fr.json [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+  "QUESTION_BASE": "Où est basé Graasp?",
+  "ANSWER_BASE": "Nous sommes basés à Lausanne, en Suisse. Nos solutions sont également hébergées en Suisse et sont conformes à la réglementation européenne, y compris le RGPD.",
+
+  "NAV_ABOUT": "À propos",
+  "NAV_PARTNERS": "Partenaires",
+  "NAV_TEAM": "Équipe",
+  "NAV_FAQ": "FAQ",
+
+  "COOKIES_ACCEPT_BUTTON_TEXT": "J'accepte",
+  "COOKIES_TEXT": "Nous utilisons des cookies et d'autres technologies de suivi pour améliorer votre expérience de navigation sur notre site Web, pour analyser le trafic de notre site Web et pour comprendre d'où viennent nos visiteurs. En naviguant sur notre site Web, vous consentez à notre utilisation des cookies et autres technologies de suivi."
```
Comment (pyphilia) [inline src/config/i18n/fr.json:55]: These translations exist in common https://github.com/graasp/graasp-translations/blob/main/src/langs/fr/common.json 
I think you can delete yours and use the right key to refer to this other translations.

**Review by pyphilia — CHANGES_REQUESTED** [review]
Great PR! I've suggested some minor changes, let me know if you have any question.

File: src/components/common/SEO.js (hunk 2 above)
Comment (juancarlosfarah) [reply src/components/common/SEO.js:24]: @pyphilia sure! Where/what do you think we should factor out?

File: src/components/sections/About.js (hunk 1 above)
Comment (juancarlosfarah) [reply src/components/sections/About.js:149]: Sure! Same as above? Where do you suggest I factor this out to?

File: src/components/sections/Faq.js (hunk 3 above)
Comment (juancarlosfarah) [reply src/components/sections/Faq.js:82]: Same as above!

File: src/components/sections/About.js [hunk 10]
```diff
@@ -95,41 +89,35 @@ const About = () => {
...
+                    components={[
+                      <a
+                        href="https://graasp.org"
+                        target="_blank"
+                        rel="noopener noreferrer"
+                      >
+                        Graasp
+                      </a>,
+                      <a
+                        href="https://desktop.graasp.org"
```
Comment (juancarlosfarah) [inline src/components/sections/About.js:104]: Factor this out too.

File: src/components/sections/About.js [hunk 11]
```diff
@@ -95,41 +89,35 @@ const About = () => {
...
-                      target="_blank"
-                      rel="noopener noreferrer"
-                    >
-                      Graasp Desktop
-                    </a>
-                    ), both online and offline.
+                  <Trans
+                    components={[
+                      <a
+                        href="https://graasp.org"
```
Comment (juancarlosfarah) [inline src/components/sections/About.js:97]: Factor out.

File: src/config/i18n/fr.json (hunk 9 above)
Comment (juancarlosfarah) [reply src/config/i18n/fr.json:55]: OK! Yes, I will do that once I move these files to translations.

**Review by juancarlosfarah — COMMENTED** [review]
@pyphilia, thanks for the review! Just some questions regarding where you want me to factor out some strings to.

**Comment by juancarlosfarah** [comment]
@pyphilia, so where do you think I should factor out those strings to? A `constants.js` file or just within the file itself?

**Comment by juancarlosfarah** [comment]
@pyphilia I made the changes. Please confirm all is OK.

File: src/components/common/SEO.js (hunk 2 above)
Comment (pyphilia) [reply src/components/common/SEO.js]: Probably all strings? url, img, twitter and facebook id?

File: src/components/sections/Faq.js (hunk 3 above)
Comment (pyphilia) [reply src/components/sections/Faq.js]: The email.

File: src/components/common/SEO.js (hunk 2 above)
Comment (pyphilia) [reply src/components/common/SEO.js]: Where I don't really, I guess it depends on the convention for gatsby.

File: src/components/common/SEO.js [hunk 12]
```diff
@@ -1,20 +1,28 @@
...
+    author: 'Graasp',
+    keywords: [
+      translateAssociation(ASSOCIATION.SEO_KEYWORD_GRAASP_ASSOCIATION),
+      translateAssociation(ASSOCIATION.SEO_KEYWORD_DIGITAL_EDUCATION),
+      'Graasp',
+      'EPFL',
+    ],
+    img: URL_SEO_IMAGE,
+    twitterId: 'graasp',
+    facebookId: 'graasp',
```
Comment (pyphilia) [inline src/components/common/SEO.js:23]: Is it the same as `SEO_DATA.facebookId`?

File: src/components/sections/Faq.js [hunk 13]
```diff
@@ -8,101 +8,111 @@ import React from 'react';
...
-            <Trans>
-              Yes! We understand that institutions and educators have varying
-              levels of experience with digital education. Please contact us at{' '}
-              <a href="mailto: contact@graasp.org">contact@graasp.org</a> to
-              learn more about how we can help you with training.
+          <FaqItem title={translateAssociation(ASSOCIATION.QUESTION_TRAINING)}>
+            <Trans
+              ns={namespaces.association}
+              components={[
+                <a href={MAILTO_CONTACT_GRAASP}>contact@graasp.org</a>,
```
Comment (pyphilia) [inline src/components/sections/Faq.js:103]: Factor out `contact@graasp.org`?

File: src/pages/404.js [hunk 14]
```diff
@@ -3,13 +3,15 @@ import React from 'react';
...
-      <p>You just hit a route that doesn&#39;t exist... the sadness.</p>
-    </Container>
-  </Layout>
-);
+function NotFoundPage() {
+  return (
+    <Layout>
+      <Container>
+        <h1>NOT FOUND</h1>
+        <p>You just hit a route that doesn&#39;t exist... the sadness.</p>
```
Comment (pyphilia) [inline src/pages/404.js:11]: No translation?

File: src/pages/page-2.js [hunk 15]
```diff
@@ -4,14 +4,16 @@ import { Link } from 'gatsby';
...
-const SecondPage = () => (
-  <Layout>
-    <Container>
-      <h1>Hi from the second page</h1>
-      <p>Welcome to page 2</p>
-      <Link to="/">Go back to the homepage</Link>
-    </Container>
-  </Layout>
-);
+function SecondPage() {
```
Comment (pyphilia) [inline src/pages/page-2.js:7]: I guess this page is not really used?

**Review by pyphilia — APPROVED** [review]
Looks good in general, I've added some minor comments, but it's good to be merged!

**Pull request merged**